workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -314,9 +319,9 @@ def _substeps(self) -> Sequence[Callable[..., Diagnostics]]:
             self._step_prephysics,
             self._compute_physics,
             self._apply_postphysics_to_physics_state,
-            self._apply_physics,
+            self.monitor("fv3_physics", self._apply_physics),
```

Comment:
feels not quite right to be hard-coding these `fv3_physics` and `python` names here, but at the same time they shouldn't be configuration options. It would be nice if the "name" used for monitoring was just the actual function name.

Reply:
I think it's okay. I don't see a way around hardcoding diagnostics names. Tying them to function names doesn't allow us to rename those functions when appropriate.